spec/paper_trail_scrapbook/user_journal_spec.rb

```diff
@@ -0,0 +1,182 @@
+require 'spec_helper'
+
+module PaperTrailScrapbook
+  RSpec.describe UserJournal do
+    let(:person) do
+      who = PaperTrail.whodunnit
+      PaperTrail.whodunnit = nil
+      p = Person.new(name: 'The Tim Man')
+      p.save!
+      PaperTrail.whodunnit = who
+      p
+    end
+
+    before do
+      PaperTrailScrapbook.config.whodunnit_class = Person
+      PaperTrail.whodunnit = person.id
+    end
+
+    let!(:book) { Book.create!(title: 'How the Grinch stole Xmas') }
+    let(:book2) { Book.create!(title: 'Green Eggs and Ham') }
+    let!(:author) { Person.create!(name: 'Dr. Seuss') }
+    let!(:target) { Authorship.create!(book: book, author: author) }
+
+    let(:format) { PaperTrailScrapbook.config.time_format }
+    let(:object) { described_class.new(person, {}) }
+    let(:subject) { object.story }
+
+    describe '#story' do
+      it 'provides a whole story' do
```

Comment:
I would introduce some contexts, extract the setup to let's and before/do to structure this entire spec better